### ISSUE TITLE
[Accessibility: ACC-264.1] Card: Ensure ARIA regions, landmarks and HTML sections are identifiable.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12567]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
FD-CARD (role=region) should have a mechanism that allows an accessible name value to be calculated.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/ef872fcf-5110-4c66-962f-2473510f3866">

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All versions

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Card](https://sap.github.io/fundamental-ngx/#/core/card)
Try to access card using screen reader. FD-CARD (role=region) does not have a mechanism that allows an accessible name value to be calculated.

#### Please provide stackblitz example(s).
You can refer ngx example. Issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@nikolay-kolarov Can you advise us in this request?

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

Yes, the role region requires an accessible name. Which means that either you set and calculate this internally and set it to aria-labelledby/aria-label (e.g. pointing to the card title) or if you do not know what is labelling the region/or the app passes this separately, you can delegate this to the application and explain that they need to provide a label of this region.

Best Regards,
Nikolay

**Comment by mikerodonnell89** [comment]
<img width="1442" alt="Image" src="https://github.com/user-attachments/assets/f7d8e810-2169-4d12-a509-a665aa1b0055" />

Cards do automatically label themselves by the card title, and furthermore can be explicitly set by the application developer using the ariaLabel input
